app/controllers/application_controller.rb

```diff
@@ -1374,12 +1374,32 @@ def find_checked_items(prefix = nil)
     end
   end
 
+  # Test RBAC on every item checked
+  # Params:
+  #   klass - class of accessed objects
   def find_checked_items_with_rbac(klass, prefix = nil)
     items = find_checked_items(prefix)
-    assert_rbac(current_user, klass, items)
+    assert_rbac(klass, items)
     items
   end
 
+  # Test RBAC in case there is only one record
+  # Params:
+  #   klass - class of accessed object
+  #   id    - accessed object id
+  def find_by_id_filtered(klass, id, options = {})
```

Comment:
I'd change this methods name to `find_item_with_rbac`:

- to stop getting comments like **" ❗️ - Line 64, Col 37 - Rails/DynamicFindBy - Use find_by instead of dynamic find_by_id_filtered."**
- to have similar name as `find_checked_items_with_rbac`, that does the same, except for 2 or more items

any ideas? @martinpovolny @PanSpagetka